### PR TITLE
Chore: use lambdas instead of anonymous classes

### PR DESCRIPTION
**Pull request by dbmalkovsky**

### CODE REVIEW

File: Project/src/test/java/com/github/lgooddatepicker/TestGithubIssues.java [hunk 1]
```diff
@@ -65,12 +65,7 @@ public void TestIssue82() throws InterruptedException {
     final ExceptionInfo exInfo = new ExceptionInfo();
     try {
       TestHelpers.registerUncaughtExceptionHandlerToAllThreads(
-          new Thread.UncaughtExceptionHandler() {
-            @Override
-            public void uncaughtException(Thread t, Throwable e) {
-              exInfo.set(t.getName(), e);
-            }
-          });
+          (t, e) -> exInfo.set(t.getName(), e));
```
Comment (WiseEagleOwl) [inline Project/src/test/java/com/github/lgooddatepicker/TestGithubIssues.java:68]: I'm guessing you are using an automatic refactoring here. That tool seems to simply use the parameter names from the anonymous class. 
When using lambda functions the parameter types are not visible anymore and the names have to be more concise to still have meaning. Sadly this cannot be done by any tools.

I would like it much better if the variable names would be `thread` and `throwable` here.

File: Project/src/test/java/com/github/lgooddatepicker/components/TestCalendarPanel.java [hunk 2]
```diff
@@ -222,23 +220,17 @@ public void TestDateHighlightAndVetoPolicy()
...
-        new DateHighlightPolicy() {
-          @Override
-          public HighlightInformation getHighlightInformationOrNull(LocalDate date) {
-            if (date.get(ChronoField.DAY_OF_MONTH) % 2 == 0) {
-              return new HighlightInformation(Color.green, Color.blue, "highlighted");
-            }
-            return null;
+        date -> {
+          if (date.get(ChronoField.DAY_OF_MONTH) % 2 == 0) {
+            return new HighlightInformation(Color.green, Color.blue, "highlighted");
```
Comment (WiseEagleOwl) [inline Project/src/test/java/com/github/lgooddatepicker/components/TestCalendarPanel.java:225]: On a similar note as my other comment I would very much prefer the paramter name `dateToHilight` here,

File: Project/src/test/java/com/github/lgooddatepicker/components/TestCalendarPanel.java [hunk 3]
```diff
@@ -222,23 +220,17 @@ public void TestDateHighlightAndVetoPolicy()
...
     CalendarPanel panel = new CalendarPanel(settings);
     settings.setVetoPolicy(
-        new DateVetoPolicy() {
-          @Override
-          public boolean isDateAllowed(LocalDate date) {
-            final int day = date.get(ChronoField.DAY_OF_MONTH);
-            return (day % 5 != 0);
-          }
+        date -> {
+          final int day = date.get(ChronoField.DAY_OF_MONTH);
```
Comment (WiseEagleOwl) [inline Project/src/test/java/com/github/lgooddatepicker/components/TestCalendarPanel.java:232]: On a similar note as my other comment I would very much prefer the parameter name `dateToVeto` here,

**Review by WiseEagleOwl — CHANGES_REQUESTED** [review]
@dbmalkovsky can you please merge the current *master* branch (containing commit 
b1dc072) into your PR as it will trigger the formatting (if needed)?

There are a few naming issues that your changes have brought to light it would be great if you could take a look. Thanks a lot.

File: Project/src/test/java/com/github/lgooddatepicker/components/TestCalendarPanel.java (hunk 3 above)
Comment (dbmalkovsky) [reply Project/src/test/java/com/github/lgooddatepicker/components/TestCalendarPanel.java]: done

File: Project/src/test/java/com/github/lgooddatepicker/components/TestCalendarPanel.java (hunk 2 above)
Comment (dbmalkovsky) [reply Project/src/test/java/com/github/lgooddatepicker/components/TestCalendarPanel.java:225]: done

File: Project/src/test/java/com/github/lgooddatepicker/TestGithubIssues.java (hunk 1 above)
Comment (dbmalkovsky) [reply Project/src/test/java/com/github/lgooddatepicker/TestGithubIssues.java]: done

**Review by WiseEagleOwl — APPROVED** [review]
Thanks for the quick response

**Pull request merged**